### PR TITLE
Prod, staging, dev release workflow files

### PR DESCRIPTION
**Pull request by herbig**
## Description

Adds workflow files for release to our three environments.

Note this will break deploy previews for the time being.  I would like to ensure this works properly, then add deploy previews in a followup PR.

## Testing

I will have to test the workflow files once they are merged, unfortunately..

1. Step one will be removing secrets from the dev site on Netlify and pushing changes and ensuring that the dev site builds as expected.
2. Then I will remove secrets from the staging site and merge dev into staging and test that that operates as expected.
3. Finally, I can remove secrets from the productions site in Netlify, and merge staging into production, and ensure that a build is NOT created.
4. Then a release tag on production should build the production release.

### CODE REVIEW

File: .github/workflows/release-netlify-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  deploy:
+    name: 'Deploy'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: jsmrcaga/action-netlify-deploy@master
+        with:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }} # auth token, generated at https://app.netlify.com/user/applications#personal-access-tokens
+          NETLIFY_SITE_ID: 'ccad1ffe-d58e-4238-bbca-cc09315855cf' # development site Netlify id (not a private identifier)
```
Comment (herbig) [inline .github/workflows/release-netlify-dev.yaml:19]: these are public ids so don't need to be in secrets.  I would rather hardcode those than manage more unnecessary secrets.

File: .github/workflows/release-netlify-dev.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          NETLIFY_DEPLOY_TO_PROD: true
+          REACT_APP_ALCHEMY_API_KEY: ${{ secrets.REACT_APP_ALCHEMY_API_KEY }}
+          REACT_APP_INFURA_API_KEY: ${{ secrets.REACT_APP_INFURA_API_KEY }}
+          REACT_APP_SENTRY_DSN: ${{ secrets.REACT_APP_SENTRY_DSN }}
+          REACT_APP_SITE_URL: ${{ secrets.REACT_APP_SITE_URL }}
```
Comment (herbig) [inline .github/workflows/release-netlify-dev.yaml:24]: these will need to be updated once @mudrila 's PR is pushed with new secrets

**Comment by Da-Colon** [comment]
Do we want to add a job to run tests that must pass before deployment? or future PR?

**Comment by herbig** [comment]
> Do we want to add a job to run tests that must pass before deployment? or future PR?

good call, but yeah I'll do that in a followup PR

**Review by Da-Colon — APPROVED** [review]
🔥

File: .github/ISSUE_TEMPLATE/config.yml
[the remark this answers is not in the document]
Comment (herbig) [reply .github/ISSUE_TEMPLATE/config.yml:1]: oh yeah you're right, I didn't even notice I had used a different extension...

File: src/constants/common.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+export const RELEASE_VERSION = '0.0.1';
```
[the remark this answers is not in the document]
Comment (herbig) [reply src/constants/common.ts:1]: ah yeah, good call

**Pull request merged**